home.html

```diff
@@ -17,42 +17,27 @@
       integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
       crossorigin="anonymous"
     />
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
 
     <title>first-capstone</title>
   </head>
   <body>
     <section class="head">
-      <div>
-        <header>
-          <nav class="navbar">
-            <div class="container-fluid">
-              <div class="navbar-header">
-                <button
-                  type="button"
-                  class="navbar-toggle"
-                  data-toggle="collapse"
-                  data-target="#myNavbar"
-                >
-                  <span class="icon-bar bg-danger"></span>
-                  <span class="icon-bar bg-danger"></span>
-                  <span class="icon-bar bg-danger"></span>
-                </button>
-                <a class="navbar-brand" href="#">WebSiteName</a>
-              </div>
-              <div class="collapse navbar-collapse" id="myNavbar">
-                <ul class="nav navbar-nav">
-                  <li class="active"><a href="#">Home</a></li>
-                  <li><a href="#">Page 1</a></li>
-                  <li><a href="#">Page 2</a></li>
-                  <li><a href="#">Page 3</a></li>
-                </ul>
-              </div>
-            </div>
-          </nav>
-        </header>
-      </div>
+      <header>
+        <nav>
+          <a class="logo" href="#">My logo</a>
+          <ul class="menu">
+            <li><a class="menuItem" href="#portfolio">Portfolio</a></li>
+            <li><a class="menuItem" href="#about">About</a></li>
+            <li><a class="menuItem" href="#contacts">Contacts</a></li>
+          </ul>
+          <button class="hamburger">
+            <i class="menuIcon material-icons">menu</i>
+            <i class="closeIcon material-icons">close</i>
+          </button>
+        </nav>
```

Comment:
- [ ] Good job following the design of the template 👍 Your page looks good and is very responsive 💯 but there is a small issue with the navbar. Your desktop navbar should follow the design which has the top short dark bar section with social media links, logout, and language links aligned to the right and the taller bottom bar with a logo and a list of page links.
- Your design:
![c2](https://user-images.githubusercontent.com/85113567/152380573-d99ed19f-1c70-4a2c-88b6-6edd8fdc8a55.PNG)
- Required design:
![c1](https://user-images.githubusercontent.com/85113567/152380656-86aa6773-d961-44fd-93f2-fb6de0dc3c60.PNG)

Reply:
Thank you for your suggestion. I missed it before. Now I have solved it and a screen has been added below:
![Screenshot 2022-02-04 at 01-25-05 first-capstone](https://user-images.githubusercontent.com/60083437/152416253-739d0988-f001-4558-8d51-179974564d37.png)